azkaban-common/src/main/java/azkaban/scheduler/Schedule.java

```diff
@@ -63,7 +67,7 @@ public Schedule(int scheduleId, int projectId, String projectName,
 
     this(scheduleId, projectId, projectName, flowName, status, firstSchedTime,
```

Comment:
Do you think we should refactor the constructor? I think there are far too many arguments.

Usually classes with so many members are built using a builder pattern or they are automatically injected using some DI framework like Guice.